dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -83,8 +83,27 @@ dom0-rpc-qubes.USB:
     - marker_start: "### BEGIN securedrop-workstation ###"
     - marker_end: "### END securedrop-workstation ###"
     - content: |
+        sd-devices sys-usb allow
         @anyvm @tag:sd-workstation deny
         @tag:sd-workstation @anyvm deny
+dom0-rpc-qubes.ensure.USBAttach:
+  file.managed:
+    - name: /etc/qubes-rpc/policy/qubes.USBAttach
+    - contents: |
+        @anyvm @anyvm ask
```

Comment:
If I understand correctly, this is to ensure the qubes.USBAttach policy is populated with default Qubes content, and the next step will ensure we apply rpc policies to SD-specific VMs?

Reply:
Exactly.